platform.js

```diff
@@ -0,0 +1,14 @@
+console.log( 'Copying .env.example to .env for the services: Started' );
+const fs = require( 'fs' );
+const path = require( 'path' );
+const { COPYFILE_EXCL } = fs.constants;
+const dir = 'packages';
+fs.readdir( dir, (err, list) => {
+    const services = list.filter( folder => folder.endsWith( '-service' ) );
+    services.forEach(service => {
+        const pathToEnv = dir + '/' + service + '/';
+        fs.copyFile( pathToEnv + '.env.example', pathToEnv + '.env', COPYFILE_EXCL, ( error ) => error );
```

Comment:
Can you add a condition here to check if a `.env` already exists?

Because, – 1: someone could have already created the .env file, or 2: someone might want to reinstall the dependencies – inn which case the post-install should not overwrite the existing .env config, which might already have all the values defined.

Reply:
@deshmukhmayur the COPYFILE_EXCL does exactly that so if you have a file which exists it skips that.
Here's the documentation I referred https://nodejs.org/api/fs.html#fs_fs_copyfile_src_dest_mode_callback

Reply:
Oh ohk, then this is fine.